### PR TITLE
Style update

### PR DESCRIPTION
**Pull request by sabineschaller**
Changes query output from camel case to snake case (makes it consistent with the spec). Also, `shared_secret` is base64 encoded instead of a buffer.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,6 @@
     "ilp-protocol-psk2": "^0.5.0",
     "ilp-protocol-stream": "^1.8.9",
     "lodash.camelcase": "^4.3.0",
-    "node-fetch": "^1.7.3"
+    "node-fetch": "^2.6.0"
```
Comment (wilsonianb) [inline package.json:16]: Was there something in particular that prompted updating `node-fetch`?

**Review by wilsonianb — APPROVED** [review]
LGTM :+1:

File: package.json (hunk 1 above)
Comment (sabineschaller) [reply package.json:16]: No, I just ran `npm-check` and that one popped up.

**Review by sharafian — APPROVED** [review]
LGTM, but make sure to tag this as a major version because it's a breaking change.

**Comment by sabineschaller** [comment]
@sharafian Please define major. 2.3.0 or 3.0.0?

**Comment by sharafian** [comment]
Major would be 3.0.0. Semantic versioning goes Major.Minor.Patch

**Pull request merged**